spring-cloud-kubernetes-client-config/src/main/java/org/springframework/cloud/kubernetes/client/config/KubernetesClientConfigDataLocationResolver.java

```diff
@@ -70,7 +70,7 @@ protected void registerBeans(ConfigDataLocationResolverContext resolverContext,
 			registerRetryBeans(configMapProperties, secretsProperties, bootstrapContext, coreV1Api, namespaceProvider);
 		}
 		else {
-			if (configMapProperties != null && configMapProperties.enableApi()) {
```

Comment:
@ryanjbaxter this was the problem... but imho, we have a bigger one. this speaks about the unit tests that we have around config data, something on my TODO list. This was caught by accident, but a test that we added very recently... I'll try to work on such unit tests in the upcoming weeks.

Reply:
Not sure i understand, this looks like you just renamed the method

Reply:
It was `isEnabled`, I renamed it to `enableApi`, which is wrong and was a type. Spent hours trying to figure it out, the renamed again ( correctly ) to `enabled`. This should have been caught by a unit test, far faster. :( Makes sense?

Reply:
but anyway, this is now ready to go. thank you